Userland/Applications/PixelPaint/CMakeLists.txt

```diff
@@ -17,6 +17,16 @@ set(SOURCES
     FilterGallery.cpp
     FilterGalleryGML.h
     FilterModel.cpp
+    Filters/BoxBlur3.cpp
+    Filters/BoxBlur5.cpp
+    Filters/Filter.cpp
```

Comment:
The Filters.cpp cmake change should be in the previous commit?

Reply:
that's right!
I cleaned it up